core/hypermodeling/src/SectionMarkerHandler.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import { IModelApp } from "@itwin/core-frontend";
-import type { AbstractToolbarProps } from "@itwin/appui-abstract";
+import type { AbstractToolbarProps } from "@itwin/core-extension";
```

Comment:
this def does not exist in core-extension; we will need to figure out what this api looks like